README.md

```diff
@@ -703,7 +703,7 @@ In the above example, the feature will be enabled for users named `Jeff` and `Al
 
 When new features are added to an application, there may come a time when a feature has multiple different proposed design options. A common solution for deciding on a design is some form of A/B testing, which involves providing a different version of the feature to different segments of the user base and choosing a version based on user interaction. In this library, this functionality is enabled by representing different configurations of a feature with variants.
 
-Variants enable a feature flag to become more than a simple on/off flag. A variant represents a value of a feature flag that can be a string, a number, a boolean, or even a configuration object. A feature flag that declares variants should define under what circumstances each variant should be used, which is covered in greater detail in the [Allocating a Variant](./README.md#allocating-a-variant) section.
+Variants enable a feature flag to become more than a simple on/off flag. A variant represents a value of a feature flag that can be a string, a number, a boolean, or even a configuration object. A feature flag that declares variants should define under what circumstances each variant should be used, which is covered in greater detail in the [Allocating a Feature's Variants](./README.md#allocating-a-features-variants) section.

```

Comment:
the reference should be Allocating Variants now.

Reply:
Given some other section names were updated, it's worth a double check if any links need updated pointing to those.

Reply:
Ok I'll double check